### ISSUE TITLE
Content not matched with text

### ISSUE THREAD

**Issue by BlueCat7** [lucidrains/DALLE2-pytorch#169]
I want to use a text as input to generate a 64x64 image, so I use a prior network and a decoder network to do inference by end to end. The prior network directly download from https://huggingface.co/nousr/conditioned-prior/tree/main/vit-b-32, and the decoder network I training on cc3m dataset. However, the content of the generated image is not matched with the text. My question is should I fine-tune the prior network and decoder network by end to end? Thanks.

**Comment by nousr** [comment]
Hey!

So a few things:

1. We were able to get reasonable results without training the prior and decoder on the same dataset, **however**, **_we did see an improvement when fine-tuning the prior on the same dateset as the decoder_**
2. Try increasing the `cond_scale` to somewhere between `3-5`, we saw a great improvement in fidelity when doing this (a better method would be to do a sweep and calculate the optimal scale using FID or some metric)
3. As always, doesn't hurt to ask and make sure you have verified that the decoder works and is sufficiently trained (i.e. can you feed it an image embedding and have it decode the image reasonably well)

If you try all of these things and they still don't work that's very valuable information! we can revisit the discussion then.

**Comment by BlueCat7** [comment]
Thanks for your detailed answers, I will carefully check it by these three steps.
